problemtools/verifyproblem.py

```diff
@@ -582,6 +582,13 @@ def __init__(self, problem):
             self.languages.append('')
         for f in glob.glob(glob_path + '[a-z][a-z].tex'):
             self.languages.append(re.search("problem.([a-z][a-z]).tex$", f).group(1))
+        attachments_path = os.path.join(problem.probdir, 'attachments')
+        if os.path.isdir(attachments_path):
+            self.attachments = os.listdir(attachments_path)
+        else:
+            self.attachments = []
+        self.debug('Adding attachments %s' % str(self.attachments))
+
```

Comment:
This (and the changes to the check) method should now be removed since the code has moved to the new class.